roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -114,6 +114,12 @@ filestore split multiple = {{ filestore_split_multiple }}
 osd op threads = {{ osd_op_threads }}
 filestore op threads = {{ filestore_op_threads }}
 filestore max sync interval = {{ filestore_max_sync_interval }}
+{% if filestore_xattr_use_omap %}
+filestore xattr use omap = {{ filestore_xattr_use_omap }}
+{% elif osd_mkfs_type == "ext4" %}
```

Comment:
That's why this starts with:

    {% if filestore_xattr_use_omap %}

Just set that variable to override the default.

Can you suggest an improvement to the comments in `main.yml` that would make this clearer? Thanks.